index.html

```diff
@@ -1917,7 +1917,12 @@ <h3>Presentations</h3>
 <a>verifiable credentials</a> is a typical use of
 <a>verifiable presentations</a>.
         </p>
-
+        <p>
+<a>Verifiable presentations</a> are intended to be extremely short-lived, and
+bound to a challenge provided by a <a>verifier</a>. Details for accomplishing
+this depend on the securing mechanism, the transport protocol, and
+<a>verifier</a> policies.
+        </p>
```

Comment:
```suggestion
<a>Verifiable presentations</a> are intended to be extremely short-lived, and
bound to a challenge provided by a <a>verifier</a>. Details for accomplishing
this depend on the securing mechanism, the transport protocol, and
<a>verifier</a> policies. Without additional requirements defined by the particular securing
mechanism or embedding protocol, a <a>Verifier</a> cannot generally assume that the author of the
<a>Verifiable Presentation</a> corresponds to the first <a>Holder</a> or <a>Subject</a> of the 
<a>Verifiable Credential</a>.
        </p>
```

Reply:
pls ignore the suggestion above, i created a separate issue.

Reply:
why? looks like a good clarification to me?

Reply:
Some tweaks to @awoie's suggestion... Fixes #1165.
```suggestion
<a>Verifiable presentations</a> are intended to be extremely short-lived, and
bound to a challenge provided by a <a>verifier</a>. Details for accomplishing
this depend on the securing mechanism, the transport protocol, and
<a>verifier</a> policies. Unless additional requirements are defined by the
particular securing mechanism or embedding protocol, a <a>Verifier</a> cannot
generally assume that the <a>Verifiable Presentation</a> has any correlation
with the presented <a>Verifiable Credentials</a> beyond preservation of their
data integrity.
        </p>
```

Reply:
I agree that with @TallTed 's tweaks from above, this PR could also close #1165.

Reply:
Verifiable Credentials are not required to have any data integrity, proof is optional.

Additionally, `proof` does not secure the bytes so a presentation could end up securing different information if different securing mechanisms are used.

I think the sentence is better without the "beyond preservation of their data integrity." part.